Makefile

```diff
@@ -1,19 +1,20 @@
 #!/usr/bin/env make
 
-GIT_ROOT:=$(shell git rev-parse --show-toplevel)
+# GNU make 3.81 or later required:
+ROOT_DIR:=$(dir $(realpath $(lastword $(MAKEFILE_LIST))))
```

Comment:
This PR has been abandoned.  But `$CURDIR` only works if you invoke `make` from the `Makefile` directory, but not if you `make -f subdir/Makefile`.

Not that we do this, but to be fully correct, the contortion above seems necessary.